easybuild/easyconfigs/d/duplex-tools/duplex-tools-0.3.1-foss-2022a.eb

```diff
@@ -0,0 +1,88 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'PythonBundle'
+
+name = 'duplex-tools'
+version = '0.3.1'
+
+homepage = 'https://github.com/nanoporetech/duplex-tools'
+description = """
+Duplex Tools contains a set of utilities for dealing with Duplex sequencing data. Tools are provided
+ to identify and prepare duplex pairs for basecalling by Dorado (recommended) and Guppy, and for
+ recovering simplex basecalls from incorrectly concatenated pairs.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+_minimap2_ver = '2.24'
```

Comment:
Why do you define this? It's only used once, in the dependencies list, right? Might as well hardcode it there then.

Reply:
damn, injecting checksums overwrote the use of it in `exts_list`